Changelog.md

```diff
@@ -329,7 +329,7 @@ Compiler Features:
 
 Bugfixes:
  * Code Generator: Fix crash when passing an empty string literal to ``bytes.concat()``.
- * Code Generator: Fix internal compiler error when calling functions bound to calldata structs and arrays.
+ * Code Generator: Fix internal compiler error when calling functions attached to calldata structs and arrays.
```

Comment:
Should I update the changelog too? I did it because in the past we agreed that updating old changelogs is fine but pointing this out just in case.

Reply:
Yes: #13307. I even specifically asked @chriseth if he thought it's ok to update old changelogs and he did think we should fix issues in them.

But yeah, I'm on the fence about it myself. Fine with removing this change.

Reply:
Changes removed from the changelog.